doc/user/pathd.rst

```diff
@@ -6,11 +6,168 @@ PATH
+In order to create a config that match the graphics used in this documentation,
+we will create a segment list (SL) called SL1 with an element for each hop and
+that element will be assigned a MPLS label.
+Then the SL1 will be used in the policy ``example1``, please note also the
+preference as in the case of multiple segment list it will be used with the
+criteria of bigger number more preference.
+Let see now the final configuration that match the graphics shown above.
+
+
+.. code-block:: frr
+
+   segment-routing
+    traffic-eng
+     segment-list SL1
+      index 10  mpls label 16001
+      index 20  mpls label 16002
+     !
+     policy color 1 endpoint 192.0.2.4
+      name example1
+      binding-sid 1111
+      candidate-path preference 100 name CP1 explicit segment-list SL1
+
+
+Explicit Segment Lists and Traffic Engineering Database (TED)
+-------------------------------------------------------------
+
+Sometimes is difficult to know the values of MPLS labels
+(adjacency changes,...).
+Based in the support of IS-IS or OSPF we can activate TED support what will
+allow pathd to resolve MPLS based in different types of segments
+(:rfc: `draft-ietf-spring-segment-routing-policy-07`). The supported types are
+Type C (prefix and local interface), Type E (prefix and algorithm),
+Type F (a pair of IP's).
+So the configuration would change to this
+
+.. code-block:: frr
+
+   segment-routing
+    traffic-eng
+     mpls-te on
+     mpls-te import ospfv2
+     segment-list SL1
+      index 10  nai prefix 10.1.2.1/32 iface 1
+      index 20  nai adjacency 10.1.20.1 10.1.20.2
+     !
+     policy color 1 endpoint 192.0.2.4
+      name example1
+      binding-sid 1111
+      candidate-path preference 100 name CP1 explicit segment-list SL1
+
+
+In this case no MPLS are provided but the pathd TED support will resolve the
+configuration provided to corresponding MPLS labels.
+
+.. note::
+	Please note the ``mpls-te`` configuration added that activate the TED
+	support and points to ``ospfv2`` so
+	the ospfv2 (:ref:`ospf-traffic-engineering`) daemon must be also
+	running and configure to export TED information.
+
+.. note::
+	It would be the same for isis (:ref:`traffic-engineering`) but in the
+	moment of writting it's not fully tested.
+
+Dynamic Segment Lists
+---------------------
+
+One of the useful options to configure is the creation of policies with
+the dynamic option. In this case based in a given endpoint the SL will be
+,first calculated, and then sended by means of PCEP protocol by the configured
+PCE.
+
+.. code-block:: frr
+
+   traffic-eng
+    !
+    pcep
+     !
+     pce PCE1
+      address ip 192.0.2.10
+     !
+     pcc
+      peer PCE1 precedence 10
+    !
+    policy color 1 endpoint 192.0.2.4
+     name example
+     binding-sid 1111
+     candidate-path preference 100 name CP2 dynamic
+
+.. note::
+	Please notes the config for the remote pce to connect to and so pathd
+	is a PCEP client by adding the pcc config.
+
```

Comment:
Please note the configuration of the remote pce which allows pathd to connect to the given PCE and act as a PCC (PCEP Client)